controllers/networkpolicy/np_controller_test.go

```diff
@@ -0,0 +1,139 @@
+package networkpolicy
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+	networkingv1 "k8s.io/api/networking/v1"
+
+	. "github.com/netobserv/network-observability-operator/controllers/controllerstest"
+	"github.com/netobserv/network-observability-operator/pkg/test"
+)
+
+const (
+	timeout                     = test.Timeout
+	interval                    = test.Interval
+	conntrackEndTimeout         = 10 * time.Second
+	conntrackTerminatingTimeout = 5 * time.Second
+	conntrackHeartbeatInterval  = 30 * time.Second
+)
+
+var (
+	getCR = func(key types.NamespacedName) *flowslatest.FlowCollector {
+		return test.GetCR(ctx, k8sClient, key)
+	}
+	cleanupCR = func(key types.NamespacedName) {
+		test.CleanupCR(ctx, k8sClient, key)
+	}
+)
+
+// nolint:cyclop
+func ControllerSpecs() {
+	const operatorNamespace = "main-namespace"
+	crKey := types.NamespacedName{
+		Name: "cluster",
+	}
+	npKey1 := types.NamespacedName{
+		Name:      constants.OperatorName,
+		Namespace: operatorNamespace,
+	}
+
+	// Created objects to cleanup
+	cleanupList := []client.Object{}
+
+	BeforeEach(func() {
+		// Add any setup steps that needs to be executed before each test
+	})
+
+	AfterEach(func() {
+		// Add any teardown steps that needs to be executed after each test
+	})
+
+	Context("Deploying as DaemonSet", func() {
+		np1 := networkingv1.NetworkPolicy{}
+		It("Should create successfully", func() {
+			created := &flowslatest.FlowCollector{
+				ObjectMeta: metav1.ObjectMeta{Name: crKey.Name},
+				Spec: flowslatest.FlowCollectorSpec{
+					Namespace:       operatorNamespace,
+					DeploymentModel: flowslatest.DeploymentModelDirect,
+					Processor: flowslatest.FlowCollectorFLP{
+						ImagePullPolicy: "Never",
+						LogLevel:        "error",
+						Advanced: &flowslatest.AdvancedProcessorConfig{
+							Env: map[string]string{
+								"GOGC": "200",
+							},
+							ConversationHeartbeatInterval: &metav1.Duration{
+								Duration: conntrackHeartbeatInterval,
+							},
+							ConversationEndTimeout: &metav1.Duration{
+								Duration: conntrackEndTimeout,
+							},
+							ConversationTerminatingTimeout: &metav1.Duration{
+								Duration: conntrackTerminatingTimeout,
+							},
+						},
```

Comment:
I don't think conn-track settings has any use here, right? I guess you can remove them